application.py

```diff
@@ -1,26 +1,36 @@
 import os
 
-from flask import Flask, render_template, request
-from flask import Flask, session
+from flask import Flask,session,request,render_template,flash,logging,redirect,url_for
 from flask_session import Session
 from sqlalchemy import create_engine
-from sqlalchemy.orm import scoped_session, sessionmaker
+from sqlalchemy.orm import scoped_session,sessionmaker
+
+from datetime import datetime as dt
+from models import user,db
+
 
 app = Flask(__name__,template_folder='Template')
 
 # Check for environment variable
 if not os.getenv("DATABASE_URL"):
     raise RuntimeError("DATABASE_URL is not set")
 
-# # Configure session to use filesystem
-# app.config["SESSION_PERMANENT"] = False
-# app.config["SESSION_TYPE"] = "filesystem"
-# Session(app)
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.secret_key="email"
+
+db.init_app(app)
+def main():
+    db.create_all()
```

Comment:
create db should be in separate file

Reply:
changes have been modified. Please check.